### PR TITLE
Add stupidtable@1.0.3 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10594
Related issue(s): # #
@cdnjs/intern3 Please help me on this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/joequery/Stupid-Table-Plugin
 * Official website (optional, not the repository): https://github.com/joequery/Stupid-Table-Plugin
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 27
   - Count of stars: 589
   - Count of forks: 153
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
I saw it cc to the author from [reference ](https://github.com/cdnjs/cdnjs/pull/10386/commits/3d3b11622bce83f4cf8ecd236c3c740ad8e0e022)  which you provided.

**Comment by hare1039** [comment]
Sorry. I mean the conversation.
> @ maruilian11 We don't need to cc author when updating the library. Thanks.

**Comment by sufuf3** [comment]
@hare1039 I modified it.
@cdnjs/intern3 Please help me on this PR. Thanks.

File: ajax/libs/stupidtable/package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "stupidtable",
   "filename": "stupidtable.min.js",
-  "version": "0.0.1",
+  "version": "1.0.2",
   "description": "A stupidly small and simple jQuery table sorter plugin",
-  "homepage": "http://joequery.github.io/Stupid-Table-Plugin/",
+  "homepage": "https://joequery.github.io/Stupid-Table-Plugin/",
```
Comment (hare1039) [inline ajax/libs/stupidtable/package.json]: @sufuf3 It seemed you combine package.json enrichment with adding auto-update config. I think you should notify package.json enrichment in commit message, or separate it into 2 commits [ref](https://github.com/cdnjs/cdnjs/pull/10014/commits). What do you think?

Comment (hare1039) [inline ajax/libs/stupidtable/package.json]: @sufuf3 It seemed you combine package.json enrichment with adding auto-update config. I think you should notify package.json enrichment in commit message, or separate it into 2 commits [ref](https://github.com/cdnjs/cdnjs/pull/10014/commits). What do you think?

Comment (extend1994) [inline ajax/libs/stupidtable/package.json]: Agree. You should separate the commits.

Comment (extend1994) [inline ajax/libs/stupidtable/package.json]: Agree. You should separate the commits.

Comment (extend1994) [inline ajax/libs/stupidtable/package.json]: Agree. You should separate the commits.

Comment (extend1994) [inline ajax/libs/stupidtable/package.json]: Agree. You should separate the commits.

**Comment by sufuf3** [comment]
@hare1039 @extend1994  I modified it.
@cdnjs/intern3 Please help me on this PR. Thanks.

**Comment by sufuf3** [comment]
ping @cdnjs/intern3

**Comment by sufuf3** [comment]
@clairetsai818 I updated this lib to v1.0.3 and maked second commit message into one line.
@cdnjs/intern3 Please help me on this PR. Thanks.

**Comment by sufuf3** [comment]
@kennynaoh Please help me on this PR. Thanks.

**Comment by pvnr0082t** [comment]
@extend1994 How;s your part?

**Comment by extend1994** [comment]
@pvnr0082t It's no problem.

**Comment by PeterDaveHello** [comment]
`Update stupidtable@1.0.3` -> `Update stupidtable to v1.0.3`, different.

**Comment by sufuf3** [comment]
@PeterDaveHello I modified the commit message, please help me review this PR, thank you.

**Pull request merged**